qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -139,7 +139,9 @@ def __init__(self,
         self._update_status_position(_status, kwargs.pop('infoQueue', None))
 
         # Properties used for caching.
+        self._result = None
```

Comment:
`self._result` could have been set on line 130 (see `job/schema`). You don't want to override it here.